dissect/target/plugins/browsers/chrome.py

```diff
@@ -1,116 +1,29 @@
-from dissect.sql import sqlite3
-from dissect.sql.exceptions import Error as SQLError
-from dissect.util.ts import webkittimestamp
-
-from dissect.target.exceptions import FileNotFoundError, UnsupportedPluginError
 from dissect.target.helpers.descriptor_extensions import UserRecordDescriptorExtension
 from dissect.target.helpers.record import create_extended_descriptor
 from dissect.target.plugin import Plugin, export
-from dissect.target.plugins.browsers.browser import (
-    GENERIC_HISTORY_RECORD_FIELDS,
-    try_idna,
-)
-
-ChromeBrowserHistoryRecord = create_extended_descriptor([UserRecordDescriptorExtension])(
-    "browser/chrome/history", GENERIC_HISTORY_RECORD_FIELDS
-)
+from dissect.target.plugins.browsers.browser import GENERIC_HISTORY_RECORD_FIELDS
+from dissect.target.plugins.browsers.chromium import ChromiumMixin
 
 
-class ChromePlugin(Plugin):
+class ChromePlugin(ChromiumMixin, Plugin):
     """Chrome browser plugin."""
 
+    HISTORY_RECORD = create_extended_descriptor([UserRecordDescriptorExtension])(
+        "browser/chrome/history", GENERIC_HISTORY_RECORD_FIELDS
+    )
     __namespace__ = "chrome"
```

Comment:
```suggestion
    __namespace__ = "chrome"

```
Last nitpick, `__namespace__` at the top and `HISTORY_RECORD` at the bottom.